sdks/python/apache_beam/typehints/trivial_inference_test.py

```diff
@@ -258,6 +258,26 @@ def testGetAttr(self):
         typehints.Tuple[str, typehints.Any],
         lambda: (typehints.__doc__, typehints.fake))
 
+  def testSetAttr(self):
+    def fn(obj, flag):
+      global glob
```

Comment:
I see we're getting this lint error: `Global variable 'glob' undefined at the module level (global-variable-undefined)`. Why are we defining a global here? Would defining `glob` outside of `def fn(obj, flag):` fix that? 

I also see there are the `store_global` and `delete_global` opcodes; were you also investigating those?

Reply:
Yes, these were correct, but I added a test for them. I've now separated that out into a separate test to be more clear (and disabled the hint warning that isn't useful here).

Reply:
Makes sense, thanks!